test/spec/modules/pubxaiAnalyticsAdapter_spec.js

```diff
@@ -28,6 +28,7 @@ describe('pubxai analytics adapter', function() {
     };
 
     let location = utils.getWindowLocation();
+    let storage = window.top['sessionStorage'];
```

Comment:
Are you intending to use [`getStorage()`](https://github.com/prebid/Prebid.js/pull/7723/files#diff-e91c97b69c5deee5b130ed490edff79b671e8f86432cad2705da283f64c9965aR24) here?

Reply:
Removed the `export` from getStorage() as it might be irrelevant.